src/main/java/lotto/domain/model/WinningNumbers.java

```diff
@@ -0,0 +1,22 @@
+package lotto.domain.model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WinningNumbers {
+    private final List<Integer> numbers;
+    private final int bonusNumber;
+
+    public WinningNumbers(List<Integer> numbers, int bonusNumber) {
+        this.numbers = numbers.stream().sorted().collect(Collectors.toList());
```

Comment:
객체지향 생활 체조 원칙 - 한 줄에 점을 하나만 사용한다!

```suggestion
        this.numbers = numbers.stream()
                .sorted()
                .toList();
```

`collect(Collectors.toList())`를 `toList()`로 쓸 수 있습니다!

Reply:
이 부분도 또 놓치고 말았네요ㅠ 다음부터 놓치지 않도록 더 신경쓰겠습니다.